testserver/binaries.go

```diff
@@ -62,7 +62,7 @@ var muslRE = regexp.MustCompile(`(?i)\bmusl\b`)
 // GetDownloadResponse return the http response of a CRDB download.
 // It creates the url for downloading a CRDB binary for current runtime OS,
```

Comment:
nit: we may want to update the comment with "`nonStable`is used only if not being given the desired version. In this case, it returns the download response and version for the latest binary."